_layouts/default.html

```diff
@@ -142,7 +142,9 @@
               <li><a href='/blog/index.html'><i class='fa fa-pencil'></i> Blog</a></li>
               <li><a href='/projects.html'><i class='fa fa-rocket'></i> Projects</a></li>
               <li><a href='/about.html'><i class='fa fa-info-circle'></i> About</a></li>
+              <li><a href='/team.html' class='contact'><i class='fa fa-comment'></i> Contact</a></li>
```

Comment:
we should have the name of the link be the same as the name of the page we're linking to. lets change it to team

also, we have not a lot of space on the upper nav. i think we'll need to collapse the links into a dropdown: https://getbootstrap.com/docs/3.3/components/#navbar